.github/workflows/refresh.yml

```diff
@@ -19,19 +19,20 @@ jobs:
       - name: Enable node
         uses: actions/setup-node@v2
         with:
-          node-version: 16.x
+          node-version: 14.x
+          git: latest
       
       # hack given that we don't have top-level dependencies
       # go into the directory and install (resets dir before next command is run)
       - name: Install dependencies
         run: cd ./.github/actions/generate-resources && yarn install --frozen-lockfile
 
+      - name: check git
+        run: echo $(git --version)
+
       - name: Run action
         uses: ./.github/actions/generate-resources
 
-      - name: Prettier clean up
-        run: ./.github/actions/generate-resources/node_modules/.bin/prettier README.md --write
-
```

Comment:
Removed this to see if there is another way to auto format the `README.md` with prettier, but still WIP.